IMPORT INTO: update document for `SHOW IMPORT` (#21610)
pingcap/docs: sql-statements/sql-statement-import-into.md

Reviewer: Because there is a dedicated doc for SHOW IMPORT GROUPS, adding a simple example of this new `GROUP_KEY` option in the Example section of this doc would be more friendly to our users.

<img width="2022" height="1130" alt="image" src="https://github.com/user-attachments/assets/7673638c-5995-41ca-ad9e-d4a8010638d1" />

For example:

####  Import a CSV file with a specific group key

```sql
IMPORT INTO mytable
FROM 's3://bucket/data.csv'
WITH GROUP_KEY='user_group';
```

Author: Good idea, done in https://github.com/joechenrh/docs/commit/caac9a6be567af672e12f098738db3303b6e1887 .

Added an `Assign a group key to the import job` example under the `IMPORT INTO ... FROM FILE` Examples section, with a link over to `SHOW IMPORT GROUPS` so users can jump to the dedicated doc for monitoring grouped jobs. 👍

| Option name | Supported data sources and formats | Description |
|:---|:---|:---|

> | `GROUP_KEY='<string>'` | All file formats | Specifies the group key for the job. The key can contain only alphanumeric characters, underscores (_), or hyphens (-), and can be up to 256 characters long. If no group key is set, the job will not belong to any group and will not appear in `SHOW IMPORT GROUPS` results. |

Reviewer: ```suggestion
| `GROUP_KEY='<string>'` | All file formats | Specifies the group key for the job. TiDB groups import jobs with the same group key, allowing you to monitor related jobs collectively using [`SHOW IMPORT GROUPS`](/sql-statements/sql-statement-show-import-group.md). The key can contain only alphanumeric characters, underscores (_), or hyphens (-), and can be up to 256 characters long. If no group key is set, the job does not belong to any group and does not appear in `SHOW IMPORT GROUPS` results. |
```

> | `GROUP_KEY='<string>'` | All file formats | Specifies an optional group key for the job. The group key does not affect the import itself; TiDB groups import jobs with the same group key so that you can monitor related jobs collectively using [`SHOW IMPORT GROUPS`](/sql-statements/sql-statement-show-import-group.md). The key can contain only alphanumeric characters, underscores (_), or hyphens (-), and can be up to 256 characters long. If no group key is set, the job does not belong to any group and does not appear in `SHOW IMPORT GROUPS` results. |

Reviewer: ```suggestion
| `GROUP_KEY='<string>'` | All file formats | Specifies an optional group key for the job. The group key does not affect the import process itself. It enables you to monitor import jobs with the same group key collectively using [`SHOW IMPORT GROUPS`](/sql-statements/sql-statement-show-import-group.md). The group key can contain only alphanumeric characters, underscores (`_`), and hyphens (`-`), up to 256 characters. If no group key is specified, the job does not belong to any group and does not appear in `SHOW IMPORT GROUPS` results. |
```

IMPORT INTO t FROM '/path/to/file.csv' WITH DETACHED;
```

#### Assign a group key to the import job

> To group related import jobs so that you can monitor them together using [`SHOW IMPORT GROUPS`](/sql-statements/sql-statement-show-import-group.md), specify the `GROUP_KEY` option:

Reviewer: ```suggestion
To monitor related import jobs collectively using [`SHOW IMPORT GROUPS`](/sql-statements/sql-statement-show-import-group.md), assign them the same group key with the `GROUP_KEY` option:
```
